varya/style-rtl.css

```diff
@@ -19,6 +19,9 @@ Underscores is distributed under the terms of the GNU GPL v2 or later.
 
 Normalizing styles have been helped along thanks to the fine work of
 Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
+
+Social Icons License: GNU GPL v2 or later.
```

Comment:
Out of curiosity, is this more of a pre-caution to include this comment, given that this is the same license as the theme itself?

Reply:
Yeah, the important bit is the source link below this line — being able to show where we got them from.